packages/x-tailwindcss/demo/src/components/xds-sliding-panel.vue

```diff
@@ -69,22 +78,17 @@
     public showButtonsOnHover!: string[];
 
     @Prop({
-      default: () => [
-        'x-sliding-panel-fade-sm',
-        'x-sliding-panel-fade-md',
-        'x-sliding-panel-fade-lg'
-      ]
+      default: () => ['', 'x-sliding-panel-at-start', 'x-sliding-panel-at-end']
     })
     public fadeSizes!: string[];
 
     public combinations = [
-      'x-sliding-panel-buttons-center x-sliding-panel-fade-sm',
-      'x-sliding-panel-buttons-center x-sliding-panel-fade-md',
-      'x-sliding-panel-buttons-outside x-sliding-panel-fade-lg',
+      'x-sliding-panel-buttons-center x-sliding-panel-show-buttons-on-hover',
+      'x-sliding-panel-buttons-outside x-sliding-panel-show-buttons-on-hover',
       // eslint-disable-next-line max-len
-      'x-sliding-panel-buttons-outside x-sliding-panel-fade-sm x-sliding-panel-show-buttons-on-hover x-sliding-panel-at-start',
+      'x-sliding-panel-buttons-outside x-sliding-panel-show-buttons-on-hover x-sliding-panel-at-start',
```

Comment:
The fade seems to be displayed on the left side of the panel when it has the class `x-sliding-panel-at-start`, and the same happens with the right side of the panel with `x-sliding-panel-at-end`.

Reply:
Good catch! When creating the PR I've modified a little bit the code from the aux branch leading to this issue.